scripts/npmDepsHash.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+newHash=$(nix-shell -p prefetch-npm-deps --run "prefetch-npm-deps ./package-lock.json")
+
+echo "$newHash" > ./npmDepsHash
```

Comment:
To avoid issues with execution do not use `/bin/sh`. Use a regular JS script, and also follow the other JS scripts in `scripts/*`. Check other repos we are doing this for inspiration, do not use `/bin/sh` for anything as it is not suitable for these repos due to context switching.

Reply:
I have converted it to a js script now in 1d7af25.